one_fm/public/js/doctype_js/employee.js

```diff
@@ -1,11 +1,54 @@
+frappe.ui.form.on('Employee', {
+	refresh: function(frm) {
+		frm.trigger('set_queries');
+		frm.trigger('custom_buttons');
+	},
+	set_queries: frm => {
+		// set bank account query
+		frm.set_query('bank_account', function() {
+			return {
+				filters: {
+					"party_type": frm.doc.doctype,
+					"party": frm.doc.name
+				}
+			};
+		});
+	},
+	custom_buttons: frm => {
+		// add custom buttons to employee doctype
+		if (!frm.is_new()){
+			// bank account
+			let bank_account_btn = document.querySelectorAll('button.btn.btn-new.btn-secondary.btn-xs.icon-btn[data-doctype="Bank Account"]');
+			if(bank_account_btn){
+					bank_account_btn[0].remove();
+			}
+			// end modify bank account link
+			// add bank account button
+			frm.add_custom_button('Bank Account', ()=>{
+				frappe.new_doc('Bank Account', {
+					party_type:frm.doc.doctype,
+					party:frm.doc.name,
+					account_name:frm.doc.employee_name,
+				})
+			}, 'Create');
```

Comment:
@mymi14s @taqi-onefm Creating bank account is not permitted for the employee. It is having a workflow, first the employee needs to request for new bank account. Or the bank account will be created form the Onboarding process.

Reply:
Ok